README.md

```diff
@@ -1,7 +1,14 @@
 RISC-V Debug Specification
 ==========================
 
-A pre-built version of the spec is included [here](https://github.com/riscv/riscv-debug-spec/blob/master/riscv-debug-draft.pdf).
+This repository contains the latest draft version of the Debug Specification,
+as well as an errata document that accompanies any ratified versions of the
+Debug Specification.
+
+The following pre-built PDFs may be of interest:
+* [0.13-DRAFT, which will likely be ratified](https://github.com/riscv/riscv-debug-spec/blob/task_group_vote/riscv-debug-draft.pdf)
```

Comment:
"likely be ratified" -> "has been submitted to the TC for ratification"?

Reply:
I think the former is more clear to people not familiar with the ratification process. Do you prefer to hedge a bit?

Reply:
I think these README edits belong with #391.

Reply:
Yes, but I wanted to mention the errata doc in the README, which isn't added until this change. I could have made changes in both and then resolve conflicts later, but this seemed much simpler.